### ISSUE TITLE
API Token field gets blanked on server restart

### ISSUE THREAD

**Issue by daveh79** [PeteGoo/tcSlackBuildNotifier#157]
Every time we restart the Team City server the API Token field in the Slack Notifications settings window is removed. Do you know why this would happen?
![2018-05-21 11_00_44-teamcity](https://user-images.githubusercontent.com/5502756/40314353-422dc78c-5ce6-11e8-8acf-b824ca68b448.png)

**Comment by daveh79** [comment]
I ran an upgrade to version 2017.2.4 this morning and it also blanked out the API Token field.

**Comment by marvin-w** [comment]
Thanks @daveh79. Do you have any logs from your server which you might be able to share with us?

This is only for the case that we can't reproduce the issue.

**Comment by daveh79** [comment]
Sure, here you go.
[teamcity-server.log](https://github.com/PeteGoo/tcSlackBuildNotifier/files/2026490/teamcity-server.log)

**Comment by marvin-w** [comment]
Thanks @daveh79.

It appears that your slack URL is broken (I'm unsure if thats connected to the mentioned issue):

`[2018-05-22 08:01:57,152]   WARN -   jetbrains.buildServer.SERVER - .slack.com 
java.net.UnknownHostException: .slack.com`

**Comment by daveh79** [comment]
I agree that the slack URL is broken but we think it's because the API Token is being removed.

**Comment by marvin-w** [comment]
Yea, could be. I'm very busy lately, I hope I can take a look in it soon but I can't promise anything.
